src/components/editor/property/OutlineHeader.jsx

```diff
@@ -43,12 +44,12 @@ const OutlineHeader = (props) => {
   return (
     <div className={groupClasses}>
       <button type="button" className="btn btn-sm btn-toggle" onClick={props.handleToggle} data-id={props.id} disabled={isAdd}>
-        <FontAwesomeIcon icon={icon} />
+        <FontAwesomeIcon id="toggle-icon" icon={toggleIcon} />
       </button>
       <strong><PropertyLabel propertyTemplate={props.property} /></strong>
       <PropertyLabelInfo propertyTemplate={ props.property } />
-      <button type="button" className="btn btn-sm btn-outline-primary btn-remove" onClick={props.handleRemoveButton} data-id={props.id}>
-        Remove
+      <button type="button" className="btn btn-sm btn-remove" onClick={props.handleRemoveButton} data-id={props.id}>
+        <FontAwesomeIcon id="trash-icon" icon={trashIcon} />
```

Comment:
id should be unique.

Reply:
HMM...the reason I did this was to get a hook on that icon for use in testing.  Maybe I can use a class instead.

Reply:
removed the need for using an ID here